src/client/src/components/abstract/sidebar/sidebarTitles/style.css

```diff
@@ -1,42 +1,60 @@
-.dropdown {
-    position: relative;
-    display:block;
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    list-style: none;
+    text-decoration: none;
+}
+.sidemenu{
     text-transform: capitalize;
-    height: 100px;
-    width: 100%;
-    margin-left: 15px;
-}
-.drop-btn{
-    background: none;
-    border:none;
-    outline: none;
+}
+.menu{
+    overflow: hidden;
+}
+.item{
+    overflow: hidden;
+}
+.btn{
+    display: block;
+    margin-top: 10px;
     color: #f9f9f9;
     text-transform: capitalize;
     font-size: 18px;
+    padding: 8px 15px;
+    text-decoration: none;
 }
-.drop-btn .icon{
-    color:#FF4800;
-    margin-right: 10px;
+.smenu{
+    overflow: hidden;
+    transition: max-height 0.3s;
 }
-.dropdown-content {
-    visibility: visible;
-    position: absolute;
+.smenu .link{
+    display: block;
+    padding: 10px 26px;
     color: #7A7575 !important;
-    padding-left: 20px;
-    z-index: 1;
     font-size: 16px;
+    margin: 4px 0;
+    position:relative;
+    text-indent: 20px;
 }
-.dropdown:hover .dropdown-content {
-   visibility: visible;
+.smenu .link:before{
+    content:"";
+    position: absolute;
+    width: 5px;
+    height: 100%;
+    background: #FF4800;
+    left:0;
+    top:0;
+    transition: 0.3s;
+    opacity: 0;
 }
-.dropdown-content .link{
-    padding-top: 12px;
-    padding-left: 15px;
-    display: block;
-    color: #7A7575 !important;
-    text-decoration: none;
+.smenu .link:hover:before{
```

Comment:
It would be better to rename className `smenu` to some name more readable like `sub-menu`

Reply:
I replace it with _sub-menu_ className